src/NuGet.Core/NuGet.Common/project.json

```diff
@@ -22,19 +22,24 @@
         "System.Core": "",

```

Comment:
@jainaashish has a change for that. There was an issue where they were not implicit for a while.